agent/resources/config/agent-log4j.properties

```diff
@@ -25,6 +25,6 @@ log4j.logger.org.apache.http.wire=ERROR
 log4j.appender.FILE=org.apache.log4j.RollingFileAppender
 log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
 log4j.appender.FILE.layout.ConversionPattern=%d{ISO8601} [%-9t] %-5p %-16c{4}:%L %x- %m%n
-log4j.appender.FILE.File=logs/go-agent.log
+log4j.appender.FILE.File=${go.agent.log.dir}/go-agent.log
```

Comment:
Is this required to do? We initialize the system property in agent bootstrapper, launcher and AgentMain. Supposing somebody using the agent rpm sets the go.agent.log.dir property, then, because we have hardcoded the /var/log/go-agent bit in all 3 log4j properties, this system property won't be used.